app/usedgoodreads/templates/index.html

```diff
@@ -4,6 +4,9 @@
     <meta charset='utf-8' />
     <title>🔖 Used Good Reads</title>
     <meta name='viewport' content='initial-scale=1,maximum-scale=1,user-scalable=no' />
+    {% if refresh %}
+    <meta http-equiv="refresh" content="3">
+    {% endif %}
```

Comment:
Perfect! :tada:

If we need anything more fancy than this, we should probably split backend and frontend properly and then just make http requests returning json.